- adds change notifications latency information (#9235)
microsoftgraph/microsoft-graph-docs: concepts/webhooks.md

> **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.

## Latency

> The following table documents the average latency between an event happening in the service and the delivery of the change notification.

Author: ```suggestion
The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.
```

| Resource | Average latency | Maximum latency |
|:-----|:-----|:-----|

> |[callRecord][] | 15 minutes | 60 minutes |

Author: ```suggestion
|[callRecord][] | below 15 minutes | below 60 minutes |
```

> |[chatMessage][] (preview) | Unknown | Unknown |

Author: ```suggestion
|[chatMessage][] (preview) | below 10 seconds | 1 minute |
```

> |[chatMessage][] (preview) | below 10 seconds | 1 minute |

Author: ```suggestion
|[chatMessage][] | below 10 seconds | 1 minute |
```

Author: related to #9372

|[contact][] | Unknown | Unknown |

> |[driveItem][] | Unknown | Unknown |

Author: ```suggestion
|[driveItem][] | below 1 minute | 5 minutes |
```

|[event][] | Unknown | Unknown |

> |[group][] | Unknown | Unknown |

Author: ```suggestion
|[group][] | below 2 minutes | 15 minutes |
```

|[conversation][] | Unknown | Unknown |

> |[list][] | Unknown | Unknown |

Author: ```suggestion
|[list][] | below 1 minute | 5 minutes |
```

|[message][] | Unknown | Unknown |

> |[alert][] | Unknown | Unknown |

Author: ```suggestion
|[alert][] | Unknown | Unknown |
|[presence][] (preview) | Unknown | Unknown |
```

> |[user][] | Unknown | Unknown |

Author: ```suggestion
|[user][] | below 2 minutes | 15 minutes |
```

> **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.

## Latency

> The following table documents the latency to expect between an event happening in the service and the delivery of the change notification.

Reviewer: ```suggestion
The following table lists the latency to expect between an event happening in the service and the delivery of the change notification.
```

| Resource | Average latency | Maximum latency |
|:-----|:-----|:-----|
|[callRecord][] | below 15 minutes | 60 minutes |
|[chatMessage][] (preview) | below 10 seconds | 1 minute |

> |[contact][] | Unknown | Unknown |

Reviewer: Just curious, does this provide value to list that we don't know what the latency is?

Author: it's a placeholder for the workloads to fill in. I'm expecting EXO to provide their values within a few weeks, and I'm working with security graph to get theirs as well.

|[driveItem][] | below 1 minute | 5 minutes |
|[event][] | Unknown | Unknown |
|[group][] | below 2 minutes | 15 minutes |
|[conversation][] | Unknown | Unknown |
|[list][] | below 1 minute | 5 minutes |
|[message][] | Unknown | Unknown |

> |[alert][] | Unknown | Unknown |

Author: ```suggestion
|[alert][] | below 3 minutes | 5 minutes |
```

|[presence][] (preview) | below 10 seconds | 1 minute |
|[user][] | below 2 minutes | 15 minutes |

> 

Author: ```suggestion

>**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time a rule takes to generate an alert from the data.
```

> **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.

## Latency

The following table lists the latency to expect between an event happening in the service and the delivery of the change notification.

| Resource | Average latency | Maximum latency |
|:-----|:-----|:-----|
|[callRecord][] | below 15 minutes | 60 minutes |
|[chatMessage][] (preview) | below 10 seconds | 1 minute |
|[contact][] | Unknown | Unknown |
|[driveItem][] | below 1 minute | 5 minutes |
|[event][] | Unknown | Unknown |
|[group][] | below 2 minutes | 15 minutes |
|[conversation][] | Unknown | Unknown |
|[list][] | below 1 minute | 5 minutes |
|[message][] | Unknown | Unknown |
|[alert][] | below 3 minutes | 5 minutes |
|[presence][] (preview) | below 10 seconds | 1 minute |
|[user][] | below 2 minutes | 15 minutes |

> >**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time it takes for a rule takes to create an alert from the data.

Author: ```suggestion
>**Note:** the latency provided for the **alert** resource is only applicable once the alert itself has been created. It does not include the time it takes for a rule to create an alert from the data.
```
